views/js/runner/provider/qti.js

```diff
@@ -273,10 +299,7 @@ define([
 
                 submit()
                  .then(updateStats)
-                 .then(computeNextMove)
-                 .catch(function(err){
-                    self.trigger('error', err);
-                 });
+                 .then(computeNextMove);
```

Comment:
I think we should keep the `catch` method. I know the promise inside the proxy is catched however, here if the promise inside `updateStats` is rejected, we won't go into the proxy's `cacth` (same for the other locations of the provider).